# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ipmiutil/hotfix.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ipmiutil-1.9.6/doc/Makefile.vanilla	2007-05-19 13:25:15.000000000 +0000
+++ ipmiutil-1.9.6/doc/Makefile	2007-05-19 13:25:31.000000000 +0000
@@ -47,5 +47,5 @@
 	${INSTALL_SCRIPT} checksel     ${datato}
 	${INSTALL_SCRIPT} wdt.sh       ${datato}
 	${INSTALL_SCRIPT} ipmi_if.sh   ${datato}
-	cd ${DESTDIR}${mandir}/man8; gzip -f *.8
+	#cd ${DESTDIR}${mandir}/man8; gzip -f *.8
 	
